style.css

```diff
@@ -102,6 +128,126 @@ a {
 }
 
 .hero_arrow-down {
-  padding: 5rem;
+  padding: 2rem;
   cursor: pointer;
 }
+
+/* Recent work styles */
+.work__heading {
+  text-align: center;
+  margin-bottom: 2rem;
+  padding-bottom: 3rem;
+}
+
+.work__heading-recent,
+.work__heading-work {
+  display: block;
+}
+
+.work__heading-work {
+  position: relative;
+}
+
+.work__heading-work::before {
+  content: "";
+  position: absolute;
+  bottom: -25px;
+  left: 50%;
+  right: 50%;
+  transform: translate(-50%, -50%);
+  height: 3px;
+  width: 15%;
+  background: var(--text-green);
+}
+
+.projects {
+  text-align: center;
+}
+
+.card {
+  border: 2px solid var(--gray);
+  margin-bottom: 1rem;
+  border-radius: 5px;
+}
```

Comment:
- [ ] Make sure you are using both grid and flex display technology of CSS. In this milestone of the project, it's required to align the projects using `display: grid` and the project of each card align using `display: flex.`

Reply:
:red_circle:  Great implementation of the grid layout :clap: :1st_place_medal: as requested by the previous reviewer. But  I think you applied it to the wrong element. The [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that this layout should be applied to the parent container. Please read the second line in the screenshot :point_down: 

![requirement](https://user-images.githubusercontent.com/26736582/200043935-074b671b-0b02-48b3-90f4-4e73a7124d9d.png)

I think according to the project requirement your Grid layout should be applied to the element with the class name projects  which is this :arrow_right: ` <div class="projects">` and the flexbox to all the elements with the class name `.card`  under the projects section